src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs

```diff
@@ -46,9 +47,11 @@ public class ForkchoiceUpdatedHandler : IForkchoiceUpdatedHandler
     private readonly ILogger _logger;
     private readonly IPeerRefresher _peerRefresher;
     private readonly ISpecProvider _specProvider;
+    private readonly bool _simulateBlockProduction;
 
     public ForkchoiceUpdatedHandler(
         IBlockTree blockTree,
+        IInitConfig initConfig,
```

Comment:
would prefer to just pass one flag at the end with default `false` rather than the whole config.